.travis.yml

```diff
@@ -15,7 +15,7 @@ notifications:
 
 matrix:
   allow_failures:
-    - os: osx
+    - os: linux
```

Comment:
Can we specify it for both linux and osx? The way you have done it now makes is so that osx must pass, which we know is a losing battle given https://github.com/travis-ci/travis-ci/issues/7304

Reply:
Done.